app/ruffle/src/main/java/rs/ruffle/MainActivity.java

```diff
@@ -89,4 +93,15 @@ protected void onCreate(Bundle savedInstanceState) {
         });
 
     }
+
+    private void RequestNoStatusBarFeature() {
+        //Hiding the status bar this way makes it see through when pulled down
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
+        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
+    }
+    private void HideActionBar() {
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null)
+            actionBar.hide();
+    }
```

Comment:
By convention, in Java in general, and in the Android APIs as well, method names are written in `halfCamelCase`.

I think I'm just gonna fix that myself after merging.

(For the future, you can maybe look into amending commits, rebasing branches to rewrite history, and force pushing... :P)

Reply:
Oh yeah, sorry about that, I keep forgetting to do that. I've been using C# for so long, got used to the capitalized methods. I even have a difficult time keeping the curly braces on the same line.